test/test.js

```diff
@@ -0,0 +1,17 @@
+var request = require('request'),
+		assert = require('assert'),
+		base_url = "http://localhost:3000/";
+
+describe("Job portal Index",function(){
+
+	describe("GET /", function(){
+
+		it("returns status code 200", function(){
+			request.get(base_url, function(err, response,body){
+				assert.equal(200,response.statusCode);
+				done();
+			});
```

Comment:
Line contains following spacing inconsistencies:
- Tabs used instead of spaces.

*Origin: SpaceConsistencyBear, Section: `cli`.*

The issue can be fixed by applying the following patch:

```diff
--- a/test/test.js
+++ b/test/test.js
@@ -10,7 +10,7 @@
 			request.get(base_url, function(err, response,body){
 				assert.equal(200,response.statusCode);
 				done();
-			});
+            });
 		});
 	});
 
```